back/spec/requests/rack_attack_spec.rb

```diff
@@ -0,0 +1,307 @@
+    end
+  end
+
+  it 'limits password reset email requests for same email to 1 in 20 seconds' do
+    headers = { 'CONTENT_TYPE' => 'application/json' }
+    create(:user, email: 'a@b.com')
+
+    post '/web_api/v1/users/reset_password_email', params: '{ "user": { "email": "a@b.com" } }', headers: headers
+    expect(status).to eq(202) # Accepted
+
+    post '/web_api/v1/users/reset_password_email', params: '{ "user": { "email": "a@b.com" } }', headers: headers
+    expect(status).to eq(429) # Too many requests
+
+    travel_to(20.seconds.from_now) do
+      post '/web_api/v1/users/reset_password_email', params: '{ "user": { "email": "a@b.com" } }', headers: headers
+      expect(status).to eq(202) # Accepted
+    end
+  end
+
+  it 'limits search requests from same IP to 15 in 20 seconds' do
+    15.times do
+      get '/web_api/v1/initiatives?search=some-random-search-term'
+    end
+    expect(status).to eq(200) # OK
+
+    get '/web_api/v1/initiatives?search=some-random-search-term'
+    expect(status).to eq(429) # Too many requests
+
+    travel_to(20.seconds.from_now) do
+      get '/web_api/v1/initiatives?search=some-random-search-term'
+      expect(status).to eq(200) # OK
+    end
+  end
+
+  it 'limits invite acceptance requests from same IP to 10 in 20 seconds' do
+    headers = { 'CONTENT_TYPE' => 'application/json' }
+
+    10.times do
+      post '/web_api/v1/invites/by_token/:token/accept',
+           params: '{ "user": { "email": "a@b.com",
+                                "first_name": "Jane",
+                                "last_name": "Doe",
+                                "password": "test1234",
+                                "token": "invalid-token" }
+                    }',
+           headers: headers
+    end
+    expect(status).to eq(401) # Unauthorized
+
+    post '/web_api/v1/invites/by_token/:token/accept',
+         params: '{ "user": { "email": "a@b.com",
+                              "first_name": "Jane",
+                              "last_name": "Doe",
+                              "password": "test1234",
+                              "token": "invalid-token" }
+                }',
+         headers: headers
+    expect(status).to eq(429) # Too many requests
+
+    travel_to(20.seconds.from_now) do
+      post '/web_api/v1/invites/by_token/:token/accept',
+           params: '{ "user": { "email": "a@b.com",
+                                "first_name": "Jane",
+                                "last_name": "Doe",
+                                "password": "test1234",
+                                "token": "invalid-token" }
+                    }',
+           headers: headers
+      expect(status).to eq(401) # Unauthorized
+    end
+  end
+
+  # ==================================================================================================================
+  # These tests are too slow to include in the CI.
+  # Un-comment to run in local dev environment, but do not push to master.
```

Comment:
You can also use `xit` for this specific purpose https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples#temporarily-pending-by-changing-%22it%22-to-%22xit%22

Reply:
I'd also favor using RSpecs skip mechanism instead of manually commenting them out. Other than that, the PR looks great Simon. :+1: